graph/src/util/lfu_cache.rs

```diff
@@ -179,7 +179,7 @@ impl<K: Clone + Ord + Eq + Hash + Debug + CacheWeight, V: CacheWeight + Default>
         })
     }
 
-    pub fn iter<'a>(&'a self) -> impl Iterator<Item = (&K, &V)> {
+    pub fn iter<'a>(&'a self) -> impl Iterator<Item = (&'a K, &'a V)> {
```

Comment:
Did `rustc` start complaining about the lifetimes here?

Reply:
Yes. Already [merged](https://github.com/graphprotocol/graph-node/pull/5725) in master as our CI updates automatically to the the latest version of rustc.